spec/scry/completion_provider_spec.cr

```diff
@@ -14,7 +14,7 @@ module Scry
       {% end %}
 
       %code = {{code}}
-      cursor_location = [%code.lines.size, %code.size]
+      cursor_location = { %code.lines.size, %code.size }
```

Comment:
the formatter will remove the spaces here :smiley:

Reply:
@bew I think `crystal tool format` doesn't work inside macros :sweat_smile:

Reply:
I have a formatter embedded in my brain, and it removes the spaces here :stuck_out_tongue: